src/main/java/core/basesyntax/interfaces/FigureDraw.java

```diff
@@ -0,0 +1,7 @@
+package core.basesyntax.interfaces;
+
+public interface FigureDraw {
+    default void drawFigure() {
+
+    }
```

Comment:
You don't need default methods
```suggestion
    void drawFigure();
```